packages/theme-wizard-website/e2e/page.spec.ts

```diff
@@ -10,3 +10,25 @@ test('page has accessibility basics', async ({ page }) => {
   // Has document language specified
   await expect.soft(page.locator('html')).toHaveAttribute('lang', 'nl-NL');
 });
+
+test('can change heading font to Courier New', async ({ page }) => {
+  await page.goto('/');
+  await page.waitForSelector('theme-wizard-app');
+
+  const select = page.locator(`theme-wizard-sidebar font-select[name="heading-font"] select`);
```

Comment:
In Playwright tests geef ik de voorkeur aan 'accessible' locators, dus bijvoorbeeld `page.getByLabel('Koppen')`